guides/join.md

```diff
@@ -0,0 +1,41 @@
+# Tham gia chơi
+
+## Tải game :id=download
+
+Đầu tiên bạn cần tải một **Minecraft Launcher** để khởi động game
+
+- Nếu bạn đã có tài khoản premium thì có thể tải game trên trang chính thức của Mojang: https://www.minecraft.net/en-us/download
+- Nếu bạn không có tài khoản premium (hoặc không biết là gì) thì hãy tải Launcher khác
+  - Bạn có thể tìm hiểu trên Google
+  - Khuyến khích dùng **TL Legacy** [tại đây](https://vk.com/page-26018968_49701456)
```

Comment:
Cái trang toàn tiếng Nga, không biết bọn nó biết đường tải không 😞 Có lẽ nên đổi thành link tải file cài đặt trực tiếp (https://tlaun.ch/installer) hoặc file jar (https://tlaun.ch/jar)

Reply:
ok, chủ yếu là muốn người chơi biết trang real ở đâu